mac

```diff
@@ -116,13 +116,10 @@ fancy_echo 'Checking on Node.js installation...'
 
 if ! brew_is_installed "node"; then
   if ! command -v nvm > /dev/null; then
-    if ! command -v n > /dev/null; then
```

Comment:
Shouldn't it still be checking for `n`, but maybe it doesn't update it. It could look something like this.?

```shell
if command -v n > /dev/null;
    fancy_echo "We recommend using `nvm` and not `n`."
    fancy_echo "Link to the guide https://@"
fi
```